tmt/steps/discover/fmf.py

```diff
@@ -557,6 +557,11 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
                 'modified-ref',
                 tmt.utils.default_branch(repository=self.testdir, logger=self._logger))
             self.info('modified-ref', modified_ref, 'green')
+            ref_commit = self.run(
+                Command('git', 'rev-parse', '--short', f"{modified_ref}"),
```

Comment:
`str(modified_ref)` would be probably easier to get - when I see an f-string, I expect it's used because it allows formatting and so on. Here it's just a fancy `str()` call :)

Reply:
fixed in 8fb02327